src/applications/personalization/profile360/components/LoadFail.jsx

```diff
@@ -12,15 +12,13 @@ export default function LoadFail({ information }) {
   return (
     <AlertBox
       isVisible
+      headline={`We can’t access your ${information} information right now.`}
       status="warning"
       content={
-        <div>
-          <h3>We can’t access your {information} information right now.</h3>
-          <p>
-            We’re sorry. Something went wrong on our end. Please refresh this
-            page or try again later.
-          </p>
-        </div>
+        <p>
+          We’re sorry. Something went wrong on our end. Please refresh this page
+          or try again later.
+        </p>
```

Comment:
This component had evidently not been updated after some site-wide alert banner updates. I had to make these changes for the alert icon not to be on a line by itself.